models/wrf_hydro/hydro_dart_py/setup.py

```diff
@@ -8,19 +8,7 @@
     url='https://github.com/NCAR/wrf_hydro_dart',
```

Comment:
James ... this URL is (now) incorrect.   That github repo was made before the entire DART was converted to git.
I believe the correct repo now is the https://github.com/NCAR/DART    (i.e. the one you this PR would merge into).